Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  HKActivitySummaryExtensions.swift
+//  SwifterSwift
+//
+//  Created by Gilbert Lo on 7/3/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+#if canImport(HealthKit)
+import HealthKit
+
+// MARK: - Properties
+@available(watchOS 2.2, *)
+public extension HKActivitySummary {
+
+    /// SwifterSwift: Check if stand goal is met
+    var isStandGoalMet: Bool {
+        return appleStandHoursGoal.compare(appleStandHours) != .orderedDescending
+    }
```

Comment:
You can make these all 1-liners.
```suggestion
    var isStandGoalMet: Bool { appleStandHoursGoal.compare(appleStandHours) != .orderedDescending }
```

Reply:
Thanks for the suggestion, updated